tracer/build/_build/Build.GitHub.cs

```diff
@@ -704,7 +707,33 @@ LabbelerConfiguration GetLabellerConfiguration()
             };
         });
 
-    Target DownloadAzurePipelineAndGitlabArtifacts => _ => _
+    Target DownloadAzurePipelineFromBuild => _ => _
+        .Unlisted()
+        .Description("Downloads the release artifacts from the specified Azure DevOps BuildId")
+        .DependsOn(CreateRequiredDirectories)
+        .Requires(() => AzureDevopsToken)
+        .Requires(() => Version)
+        .Requires(() => AzureDevopsBuildId)
+        .Executes(async () =>
+        {
+            // Connect to Azure DevOps Services
+            var connection = new VssConnection(
+                new Uri(AzureDevopsOrganisation),
+                new VssBasicCredential(string.Empty, AzureDevopsToken));
+
+            // Get an Azure devops client
+            using var buildHttpClient = connection.GetClient<BuildHttpClient>();
+
+            int buildId = int.Parse(AzureDevopsBuildId);
+            BuildArtifact artifact = await DownloadArtifactsFromConsolidatedPipelineBuild(buildHttpClient, buildId, $"{FullVersion}-release-artifacts");
```

Comment:
nitpicking, those aren't release artifacts anymore :) Feel free to ignore

```suggestion
            BuildArtifact artifact = await DownloadArtifactsFromConsolidatedPipelineBuild(buildHttpClient, buildId, $"{FullVersion}-build-artifacts");
```

Reply:
👼🏼   😈
  Zach

I'll go ahead and leave this as-is 😆